### DISCUSSION TITLE
What is your favorite J-pop from the 1970s to the present?

### DISCUSSION THREAD

**Discussion by Ninjia315** [NAIST-SE/AttractingContributors#1103] [2024 Attraction Contributions]
Hi, thank you for reading my post!
I want to collect all genres of J-pop from the 1970s to the present. 
Please help me add you favorite J-pop song in **List of hobby**. 
You can write in English and Japanese! 
If you don't mind, I would greatly appreciate it if you could also specify singer's name!

If you are interested, please **click here**! https://github.com/Ninjia315/AttractingContributors-Favorite_Japanese_songs

**Comment by Canppchk** [comment]
I submitted a Yuuri 「ドライフラワー」 (Dried Flower) Lyrics add-on in your list.
could you help me with my project too? Thanks!!
https://github.com/Canppchk/AttractingContributors-Ramen
